test/fixtures/facebook-1.json

```diff
@@ -1,52 +1,37 @@
-      "source_id": "251841205227",
-      "post_id": "251841205227_10154221785975227",
-      "updated_time": 1401461816,
-      "created_time": 1401461816,
-      "actor_id": "251841205227",
-      "permalink": "https://www.facebook.com/photo.php?fbid=10154221776000228&set=a.261947725227.302128.251841205227&type=1",
-      "message": "The ACC Champion Georgia Tech Yellow Jackets take on Washington today in the Oxford,MS regional at 4:00 p.m. \n\nThe Jackets have played in 28 of the last 30 NCAA baseball tournaments. Can we get a #GoJackets??? -- http://c.gatech.edu/1kcD0c6"
-    }, {
-      "source_id": "251841205227",
-      "post_id": "251841205227_10154233081715228",
-      "updated_time": 1401655358,
-      "created_time": 1401622269,
-      "actor_id": "251841205227",
-      "permalink": "https://www.facebook.com/photo.php?fbid=10154233081380228&set=a.261947725227.302128.251841205227&type=1",
-      "message": "Bioengineers at Georgia Tech are developing a better way to rebuild and repair broken bones from inside the human body. -- http://c.gatech.edu/1onyDu"
-    }]
-  }, {
-    "name": "authors",
-    "fql_result_set": [{
-      "id": "251841205227",
+[
+  {
+    "id": 123123123143,
+    "from": {
+      "id": 23232323,
       "name": "Georgia Tech"
-    }, {
-      "id": "2518412052271",
-      "name": "Test User 1"
-    }]
-  }, {
-    "name": "comments",
-    "fql_result_set": [{
-      "fromid": "2518412052271",
-      "time": 1401655358,
-      "text": "Amazing!!!!",
-      "text_tags": [],
-      "post_id": "251841205227_10154233081715228",
-      "id": "251841205227_10154233081715228_10154234351525228",
-      "likes": 0,
-      "user_likes": false,
-      "post_fbid": "10154234351525228"
-    }]
-  }]
-}
+    },
+    "message": "The ACC Champion Georgia Tech Yellow Jackets take on Washington today in the Oxford,MS regional at 4:00 p.m. \n\nThe Jackets have played in 28 of the last 30 NCAA baseball tournaments. Can we get a #GoJackets??? -- http://c.gatech.edu/1kcD0c6",
+    "link": "https://www.facebook.com/photo.php?fbid=10154233081380228&set=a.261947725227.302128.251841205227&type=1",
+    "created_time": "2015-09-09T20:43:14+0000",
+    "type": "link"
+  },
+  {
+    "id": 123143123123,
+    "from": {
+      "id": 23232323,
+      "name": "Georgia Tech"
+    },
+    "message": "Bioengineers at Georgia Tech are developing a better way to rebuild and repair broken bones from inside the human body. -- http://c.gatech.edu/1onyDu",
+    "link": "https://www.facebook.com/photo.php?fbid=10154233081380228&set=a.261947725227.302128.251841205227&type=1",
+    "created_time": "2015-09-09T20:43:14+0000",
+    "type": "link",
+    "comments": {
+      "data":[
+        {
+          "id": 122122123123,
+          "from": {
+            "id": 23232323,
+            "name": "Test User 1"
+          },
+          "message": "Amazing!!!",
+          "created_time": "2015-09-09T20:43:14+0000"
+        }
+      ]
+    }
+  }
+]
```

Comment:
Great that you've updated these JSON fixtures to match what the Graph API is returning. I would suggest you double and triple check that this is what is actually being returned as if the fixtures are off then the tests are misleading.